messages.php

```diff
@@ -35,6 +35,7 @@
                     </tr>
                     </thead>
                 </table>
+                <p>Note: If errors are shown, use the tab on the left to generate a debug log, which will show a more thorough Pi-hole evaluation.</p>
```

Comment:
Maybe include a link here?

```suggestion
                <p>Note: If errors are shown, you can <a href="debug.php">generate a debug log</a>, which will do a thorough Pi-hole evaluation.</p>
```

Reply:
Good idea.